src/scheduler.js

```diff
@@ -155,7 +154,7 @@ function run(gatherers, options) {
           {traceContents: tracingData.traceContents},
           {frameLoadEvents: tracingData.frameLoadEvents});
 
-      const artifacts = flattenArtifacts(unflattenedArtifacts);
+      let artifacts = flattenArtifacts(unflattenedArtifacts);
```

Comment:
Ah I was modifying `artifacts` at some point but eventually decided against it. Will fix.